### DISCUSSION TITLE
DF55F3C4-08AA-46BA-9183-38AC8C2D16F5

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#27] [General]
# Kepler-452 (by SpaceExplorer)

Kepler-452 is a star similar to the Sun, though slightly more evolved, with a likely super-Earth in the habitable zone. The planet, Kepler-452b, was validated in 2015, but further research in 2018 and 2019 failed to confirm the planet with >99% confidence.

https://celestia.mobi/addon?item=DF55F3C4-08AA-46BA-9183-38AC8C2D16F5&lang=en

**Comment by Dirgham2015** [comment]
:I

**Reply by orangeandfriends700-dev** [reply]
What?

**Comment by Kendallcoolspace** [comment]
Why does it have rings

**Reply by DaveBowman2001** [reply]
I think it was based on SpaceEngine's [depiction](https://www.reddit.com/r/spaceengine/comments/12e50qi/hello_from_kepler452b/) of the planet
![images](https://github.com/celestiamobile/comments/assets/95486841/7bbd3f78-c75f-4da0-bd63-2e848a9119fb)

**Comment by ilyushinil594-hue** [comment]
I think through this Habitable planet have a rings like Saturn

![hello-from-kepler-452b-v0-h9f1z96j7dsa1](https://github.com/user-attachments/assets/6df9760a-8be1-4813-8ef1-87fc81402a4b)
